tests/periph/uart/main.c

```diff
@@ -401,7 +402,9 @@ static const shell_command_t shell_commands[] = {
 #endif
     { "send", "Send a string through given UART device", cmd_send },
     { "test", "Run an automated test on a UART with RX and TX connected", cmd_test },
-    { "nl", "Set line ending", cmd_newline },
+    { "eol_cr", "Set CR as the end-of-line for send", cmd_eol_cr },
+    { "eol_crlf", "Set CRLF as the end-of-line for send", cmd_eol_crlf },
+    { "eol_lf", "Set LF as the end-of-line for send (default)", cmd_eol_lf },
```

Comment:
I'm fine with removing that hack to save a few bytes of RAM, but why drop the `nl` command? I found that a bit cleaner that three separate commands just to set the line ending.

Reply:
The reason I did it with three commands is to have more documentation in the "help" overview.

"nl" does explain much, neither does "Set line ending".

Reply:
I like the more verbose way of doing it. Do you want to resolve this or ask for changes @benpicco?

Reply:
I don't have strong feelings about this, if you like it more this way go for it.